pkg/chunk/schema.go

```diff
@@ -502,9 +578,24 @@ func (v6Entries) GetChunksForSeries(_ Bucket, _ []byte) ([]IndexQuery, error) {
 type v9Entries struct {
 }
 
-func (v9Entries) GetWriteEntries(bucket Bucket, metricName model.LabelValue, labels model.Metric, chunkID string) ([]IndexEntry, error) {
+func (e v9Entries) GetWriteEntries(bucket Bucket, metricName model.LabelValue, labels model.Metric, chunkID string) ([]IndexEntry, error) {
+	labelEntries, err := e.GetLabelWriteEntries(bucket, metricName, labels, chunkID)
+	if err != nil {
+		return nil, err
+	}
+
+	chunkEntries, err := e.GetChunkWriteEntries(bucket, metricName, labels, chunkID)
+	if err != nil {
+		return nil, err
+	}
+
+	entries := append(labelEntries, chunkEntries...)
+
+	return entries, nil
+}
```

Comment:
Make this also return ErrNotSupported

Reply:
Are you sure? :) Everything but this returns `ErrNotSupported`